src/explore-education-statistics-admin/src/pages/release/data/components/DataFileUploadForm.tsx

```diff
@@ -37,6 +39,10 @@ function baseErrorMappings<FormValues extends DataFileUploadFormValues>(
           DataZipFileDoesNotContainCsvFiles:
             'ZIP file does not contain any CSV files',
           DataFilenameNotUnique: 'Choose a unique ZIP data file name',
+          DataZipFilenameTooLong: `Maximum ZIP data filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          DataFilenameTooLong: `Maximum data filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          MetaFilenameTooLong: `Maximum metadata filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          DataZipContentFilenamesTooLong: `Maximum data and metadata filenames cannot exceed ${MAX_FILENAME_SIZE} characters`,
```

Comment:
I think it might be possible to validate this on the frontend as well via Formik's `validationSchema`? I'm not sure though, and to be honest I'm fine if we leave it as is.

Reply:
There's a task being worked on at the moment to gradually replace Formik across the codebase, so on this occasion it's better to leave as-is to avoid wasted effort.